platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java

```diff
@@ -434,6 +436,14 @@ public void setWorkingIndicatorEnabled(boolean enabled) {
 		JS_call("setWorkingIndicatorEnabled", enabled);
 	}
 
+	@Override
+	public void setAlternateResourceOnError(DomImage image, IResource resource) {
+
+		Objects.requireNonNull(image);
+		Objects.requireNonNull(resource);
+		addJavascript("%s.onerror=function(){%s.src=%s;%s.onerror=null;};", image, image, getResourceUrl(resource), image);
```

Comment:
This approach, while certainly functional (pun intended), appears uncommon. Shouldn't we add a feature to the TypeScript code/API, instead of injecting JS code?

Reply:
I though about it, but in the end it would only make things more complicated, IMO

Reply:
Well, we could as well change this later-on. Okay for now.